compiler/noirc_evaluator/src/brillig/brillig_ir/procedures/array_copy.rs

```diff
@@ -88,7 +88,7 @@ pub(super) fn compile_array_copy_procedure<F: AcirField + DebugToString>(
 }
 
 /// The message to print when copying an array.
-const ARRAY_COPY_COUNTER_MESSAGE: &str = "Total arrays copied: {}";
+const ARRAY_COPY_COUNTER_MESSAGE: &str = "Total arrays copied in";
```

Comment:
Does it make sense to inline this constant into the `format!` call? It's only used there.

Reply:
Yeah I guess before it was used for also fetching the length. I'm just going to inline it.